src/core.js

```diff
@@ -164,7 +164,9 @@ extend( QUnit, {
 	stack: function( offset ) {
 		offset = ( offset || 0 ) + 2;
 		return sourceFromStacktrace( offset );
-	}
+	},
+
+    onError
```

Comment:
Oops, need to tabify this line. I can't wait until we can get ESLint indent working...